src/components/Intro/Intro.js

```diff
@@ -0,0 +1,45 @@
+import styles from "./Intro.module.css";
+
+function Intro() {
+  return (
+    <section className={styles.wrapper}>
+      <title className={styles.title}>Чего мы там не видели?</title>
+      <p className={styles.Introtext}>
```

Comment:
use the pattern to name it like `introList` it will help you to `separateOneWorkFromEnotherIfTheClassnameEnormouslyBig`

Same for the rest in the file

Reply:
done